geoips/commandline/log_setup.py

```diff
@@ -34,7 +34,8 @@ def log_with_emphasis(print_func, *messages):
         The messages to be logged with emphasis
     """
     wrapped_messages = []
-    messages = filter(lambda s: len(s) > 0, messages)
+    # Cast as a string just in case something other than a string exists in messages
+    messages = filter(lambda s: len(s) > 0 or len(str(s)) > 0, messages)
```

Comment:
I still think this will fail if any of the messages in ``*messages`` don't have a ``len`` attribute. For example, if a ``TypeError`` was raised when ``len(s)`` was called, then ``len(str(s))`` would never occur because the error was raised, correct?

Reply:
You're right. 😅 I think a try, except catch is probably the way then. I can add that